packages/ra-core/src/sideEffect/admin.ts

```diff
@@ -13,7 +14,11 @@ import undo from './undo';
 /**
  * @param {Object} dataProvider A Data Provider function
  */
-export default (dataProvider, authProvider, i18nProvider) =>
+export default (
+    dataProvider: DataProvider,
+    authProvider: AuthProvider | null,
+    i18nProvider: I18nProvider
```

Comment:
Doesn't it accept `null` too ?

Reply:
no, there is a defaulti18nProvider (english) if none is passed